Resources/Server Info/Guidebook/Science/AnomalousResearch.xml

```diff
@@ -0,0 +1,41 @@
+<Document>
+# Anomalous Research
+
+Anomalous Research is a subdepartment of Science focused around the containment and manipulation of various unique anomalies.
+
+## Equipment
+<Box>
+<GuideEntityEmbed Entity="AnomalyScanner"/>
+<GuideEntityEmbed Entity="MachineAnomalyVessel"/>
+<GuideEntityEmbed Entity="MachineAPE"/>
+</Box>
+
+Scanners, vessels, and A.P.E.s are all used in the containment and observation of anomalies.
+
+<Box>
+<GuideEntityEmbed Entity="MachineAnomalyGenerator"/>
+</Box>
+
+The anomaly generator can produce a random anomaly somewhere on the station at the cost of plasma.
+
+## Anomalies
+Anomalies are static objects that appear on the station. They cannot be moved, and must be worked around, no matter their location.
+
+<Box>
+<GuideEntityEmbed Entity="AnomalyPyroclastic"/>
+</Box>
+
+Each anomaly has 3 main stats, 2 of which can be observed through anomaly scans:
+
+- [color=#a4885c]Severity:[/color] Shown at the top of scans as a percentage. This is a numerical value corresponding to the danger of an anomaly.
+- [color=#a4885c]Stability:[/color] Shown next to severity as either decaying, stable, or growing; this is how frequently and widespread the effects will be.
```

Comment:
The location in the UI is obvious when opening the UI, so that probably doesn't really need to be specified here.